spec/features/cases/foi/case_listing/incoming_cases_spec.rb

```diff
@@ -5,44 +5,62 @@
   given(:press_officer) { find_or_create :press_officer }
   given(:private_officer) { find_or_create :private_officer }
 
-  given(:assigned_case) { create :assigned_case,
-                                 created_at: 1.business_days.ago,
-                                 identifier: 'assigned_case' }
-  given(:fresh_assigned_case) { create :assigned_case,
-                                       identifier: 'fresh_assigned_case' }
+  given(:assigned_case) do
+    Timecop.freeze(Date.new(2020, 8, 19)) do
```

Comment:
Just thinking - we are making multiple calls with the same date, maybe we could wrap everything and then call it once like this using `before` and `after` calls:

```ruby
describe "some set of tests to mock" do
  before do
    Timecop.freeze(Time.local(1990))
  end

  after do
    Timecop.return
  end

  it "should do blah blah blah" do
  end
end
```

Reply:
More info: https://github.com/travisjeffery/timecop

Reply:
I think we have that explicitly turned off 🤔 

Let's try it out sometime